WordPress/wordpress-playground
# Website: Remove "small window mode"

Playground was initially displayed in a small window taking only a part of the screen. This mode was then made optional, with the full screen becoming the default. Looking back, the small window mode doesn't seem to be useful for anything – let's remove it.

This change was inspired by @sirreall who reported a slight UX issue with CSS transitions that was caused by still supporting the small window mode:

https://github.com/WordPress/wordpress-playground/pull/824#discussion_r1648576896

Closes https://github.com/WordPress/wordpress-playground/issues/1534

 ## Testing instructions

* Open Playground
* Confirm all three buttons in the upper left corner are gray and non-interactive
* Confirm that Playground is resized together with the window, not in a delayed way as on the video below

https://github.com/WordPress/wordpress-playground/assets/841763/a201b16f-0e4d-4a52-bd93-34b6f7bf40d9
